core/templates/pages/learner-dashboard-page/learner-dashboard-page.component.html

```
@@ -82,6 +82,17 @@
             {{ LEARNER_DASHBOARD_SECTION_I18N_IDS.PROGRESS | translate }}
           </button>
         </li>
+        <li *ngIf="isCertificateAssessmentEnabled()">
+          <button aria-label="Go to my certificates tab"
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)